schedule/yast/opensuse/installer_extended/installer_extended.yaml

```diff
@@ -29,3 +30,8 @@ conditional_schedule:
     BACKEND:
       qemu:
         - installation/language_keyboard/switch_keyboard_layout
+  access_beta_distribution:
+    BETA:
+      1:
+        - installation/validate_beta_message
+        - installation/access_beta_distribution
```

Comment:
I guess we will need to set beta_text here too, right?

Reply:
Fixed and started a new VR. Good catch:)

Reply:
It passed through the step: https://openqa.opensuse.org/tests/1926714